index.js

```diff
@@ -1,83 +1,32 @@
-var commentOnOpenedPullRequest = require('./lib/commentOnOpenedPullRequest');
 
+var checkWebHook = require('./lib/checkWebHook');
 var dateLog = require('./lib/dateLog');
+var postToGitHub = require('./lib/postToGitHub');
 var Settings = require('./lib/Settings');
 
 Settings.loadRepositoriesSettings('./config.json')
-.then(function (repositoryNames) {
-    dateLog('Loaded settings successfully');
-    var webHookHandler = gitHubWebHook({
-        path: Settings.listenPath,
-        secret: Settings.secret
-    });
-
-    var app = express();
-    app.use(bodyParser.json());
-    app.use(webHookHandler);
-
-    repositoryNames.forEach(function (repositoryName) {
-        dateLog('Listening to ' + repositoryName);
-        webHookHandler.on(repositoryName, function (event, jsonResponse) {
-            dateLog('Received event to repository: ' + repositoryName);
-            dateLog('event: ' + event);
-            dateLog('jsonResponse: ' + jsonResponse);
+    .then(function () {
+        dateLog('Loaded settings successfully');
 
-            var promise = Promise.resolve();
-            var repositorySettings = Settings.repositories[repositoryName];
-            var headers = {
-                'User-Agent': 'cesium-concierge',
-                Authorization: 'token ' + repositorySettings.gitHubToken
-            };
-            var checkChangesMd = repositorySettings.checkChangesMd;
+        var app = express();
+        app.post(Settings.listenPath, bodyParser.json(), checkWebHook, postToGitHub);
 
-            if ((event === 'issues' || event === 'pull_request') && jsonResponse.action === 'closed' &&
-                repositorySettings.remindForum) {
-                promise = promise.then(function () {
-                    dateLog('Calling commentOnClosedIssue');
-                    return commentOnClosedIssue(jsonResponse, headers);
-                });
-            } else if (event === 'pull_request' && jsonResponse.action === 'opened' &&
-                (defined(repositorySettings.thirdPartyFolders) || checkChangesMd)) {
-                promise = promise.then(function () {
-                    dateLog('Calling commentOnOpenedPullRequest');
-                    return commentOnOpenedPullRequest(jsonResponse, headers, repositorySettings.thirdPartyFolders,
-                        checkChangesMd);
-                });
-            }
-
-            promise.then(function (res) {
-                if (!defined(res)) {
-                    dateLog('GitHub request did not match any events the server is listening for');
-                    return;
-                }
-                dateLog('GitHub API returned with statusCode: ' + res.statusCode);
-                dateLog('and statusMessage: ' + res.statusMessage);
-            }).catch(function (e) {
-                dateLog('Got an error: ' + e);
-            });
+        // Handle errors
+        app.use(function (err, req, res, next) { // eslint-disable-line no-unused-vars
```

Comment:
seems OK to remove.

When run in production mode, however, the errors won't have a stack trace.